.github/workflows/ci.yml

```diff
@@ -51,12 +51,16 @@ jobs:
         id: checkout
         uses: actions/checkout@v4
 
-      - name: Test Local Action
+      - name: Test Display All Mutations
         id: test-action
         uses: ./
         with:
-          who-to-greet: GitHub
+          file-path: ./mutations.xml
+
+      - name: Test Display Survives Only
+        id: test-action
```

Comment:
This line fails in linter because it has the same id `test-action` as line 55

Reply:
🤦 Thanks!